From 4bc8cd6c5427665af89f232dce43f0375db3bb0b Mon Sep 17 00:00:00 2001
From: Aleix Pol <aleixpol@kde.org>
Date: Tue, 7 Mar 2023 03:12:48 +0100
Subject: [PATCH] input: Make sure input backends are initialised when the
 workspace is set up

The outputs already present upon workspace setup wouldn't signal. This
was easily triggered running a standalone kwin session on a tty, it
would manifest with the touchscreen not working complaining that it
didn't have an output assigned yet.

BUG: 466721
---
 src/input.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/input.cpp b/src/input.cpp
index 163565570aa..e14e9328551 100644
--- a/src/input.cpp
+++ b/src/input.cpp
@@ -2636,6 +2636,7 @@ void InputRedirection::setupWorkspace()
 
         setupTouchpadShortcuts();
         setupInputFilters();
+        updateScreens();
     }
 }
 
-- 
GitLab

